modules/python/src2/cv2.cpp

```diff
@@ -1891,7 +1891,6 @@ static int convert_to_char(PyObject *o, char *dst, const ArgInfo& info)
 
 
 #include "pyopencv_generated_enums.h"
-#include "pyopencv_custom_headers.h"
```

Comment:
@alalek @mshabunin Is it OK to move this header a little lower ? 
It would gives an opportunities to use code generated by `CVPY_TYPE`

Reply:
No, it's impossible because `CVPY_TYPE` can't use types defined in corresponding files for every module like `pyopencv_gapi` and `pyopencv_dnn`

Ok, I have to create a new file or put this code to `cv2.cpp`

Reply:
But then it will be impossible to generate code for `using` and `classes` defined in custom headers, on the other hand no one custom header use this

It definitely requires additional header after `CVPY_TYPE` or putting code in `cv2.cpp`